Changelog.md

```diff
@@ -1,5 +1,9 @@
 ## 1.3.2 - UNRELEASED
 
+### Added
+
+-   Support for Apple silicon
```

Comment:
```suggestion
-   Support for Apple silicon.
```
Why not mention M1 here?

Reply:
I am getting confused ..... is M1 relay the name .... as we will support M1 M2 M3 .......